src/constants.jl

```diff
@@ -34,3 +34,8 @@ const OFFSET_TT_TAI = 32.184
 
 const MOD_JD_77 = 43144.0
 const ELG = 6.969290134e-10
+const DJM = 365250.0
```

Comment:
Should this be defined in terms of `DAYS_PER_YEAR`.  I guess these are days per millennium, maybe the constant can be given a more meaningful name.

Reply:
👍